src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -203,19 +217,53 @@ public static void main(String[] args) {
         return httpPromise.future();
     }
 
-    private static String getFilePath(String arg) {
-        String[] confFileArg = arg.split("=");
-        String path = "";
-        if (confFileArg[0].equals("--config-file")) {
-            if (confFileArg[1].startsWith(File.separator)) {
-                // absolute path
-                path = confFileArg[1];
-            } else {
-                // relative path
-                path = System.getProperty("user.dir") + File.separator + confFileArg[1];
-            }
-            return path;
+    /**
+     * Return a JmxCollectorRegistry instance with the YAML configuration filters provided
+     * through the command line or configured without any filters
+     *
+     * @param commandLine command line parameters containing the path to the JMX YAML configuration
+     * @return  JmxCollectorRegistry instanc
+     * @throws MalformedObjectNameException
+     * @throws IOException
+     */
+    private static JmxCollectorRegistry getJmxCollectorRegistry(CommandLine commandLine)
+            throws MalformedObjectNameException, IOException {
+        if (commandLine.hasOption("jmx-metrics-config")) {
+            return new JmxCollectorRegistry(new File(getFilePath(commandLine.getOptionValue("jmx-metrics-config"))));
+        } else {
+            return new JmxCollectorRegistry("");
         }
-        return null;
+    }
+
+    /**
+     * Generate the command line options
+     *
+     * @return command line options
+     */
+    private static Options generateOptions() {
+
+        Option configFileOption = Option.builder()
+                .required(true)
+                .hasArg(true)
+                .longOpt("config-file")
+                .desc("Configuration file with bridge parameters")
+                .build();
+
+        Option jmxMetricsFileOption = Option.builder()
+                .required(false)
+                .hasArg(true)
+                .longOpt("jmx-metrics-config")
+                .desc("JMX Kafka metrics YAML configuration file")
+                .build();
+
+        Options options = new Options();
+        options.addOption(configFileOption);
+        options.addOption(jmxMetricsFileOption);
+        return options;
+    }
+
+    private static String getFilePath(String arg) {
```

Comment:
If it's returning an absolute file path, why not call this function `absoluteFilePath`

I know Tom Bentley in the past has mentioned we shouldn't use `get` in a method name if we aren't actually retreiving a field value in a javabean fashion, so we can trim this down to `absoluteFilePath(String path)` and the intent and purpose of the function is clear I think